CodeEdit/Features/StatusBar/ViewModels/StatusBarViewModel.swift

```diff
@@ -25,7 +36,7 @@ class StatusBarViewModel: ObservableObject {
     /// - **1**: Debugger
     /// - **2**: Output
```

Comment:
```suggestion
```
It seems that this explanation is no longer needed because of the enum's `.terminal`, `.debugger`, ...